src/controls/SessionSheet.tsx

```diff
@@ -54,6 +54,20 @@ const SessionSheet: FC<SessionSheetProps> = ({
     editSession(reference, session);
   };
 
+  const shouldShowDescription = () => {
```

Comment:
This should be memoized instead of a function, calling this a ton of times is unnecessary and a performance hit.